Corona-Warn-App/src/main/res/layout/fragment_settings_background_priority.xml

```diff
@@ -57,34 +57,84 @@
                     app:layout_constraintStart_toStartOf="parent"
                     app:layout_constraintTop_toTopOf="parent" />
 
-                <include
-                    android:id="@+id/settings_switch_row_background_priority"
-                    layout="@layout/include_settings_switch_row"
-                    android:layout_width="@dimen/match_constraint"
+                <androidx.constraintlayout.widget.ConstraintLayout
+                    android:id="@+id/settings_row_background_priority"
+                    android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:layout_marginTop="@dimen/spacing_small"
-                    app:enabled="@{!settingsViewModel.isBackgroundPriorityEnabled()}"
                     app:layout_constraintEnd_toEndOf="parent"
                     app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintTop_toBottomOf="@+id/settings_background_priority_header_details"
-                    app:showDivider="@{true}"
-                    app:status="@{settingsViewModel.isBackgroundPriorityEnabled()}"
-                    app:statusText="@{FormatterSettingsHelper.formatStatus(settingsViewModel.isBackgroundPriorityEnabled())}"
-                    app:subtitle="@{@string/settings_background_priority_title}" />
+                    app:layout_constraintTop_toBottomOf="@+id/settings_background_priority_header_details">
+
+                    <androidx.constraintlayout.widget.ConstraintLayout
+                        style="@style/row"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:focusable="true"
+                        app:layout_constraintEnd_toEndOf="parent"
+                        app:layout_constraintStart_toStartOf="parent"
+                        app:layout_constraintTop_toTopOf="parent">
+
+                        <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
ConstraintLayout within a ConstraintLayout should not be necessary.

Reply:
this is taken directly from the `include_settings_switch_row` include and is (i think) used to apply the row style to some of the children of the parent element

Reply:
And now you can remove the nested ConstraintLayout, too @kolyaopahle